website_forum_subscription/readme/USAGE.rst

```diff
@@ -0,0 +1,7 @@
+To use this module, you need to:
+
+#. Go to your website /forum page.
+#. If there are several forums, you'll find a subscription button in every card. Click
+   on it to subscribe or unsubscribe depending on the state of your forum subscription.
+#. In the main forum page, you'll find the same button on the top of the forum panel.
```

Comment:
#. In the main forum page, you'll find the same button on top of the forum panel.